FBSnapshotTestCase/FBSnapshotTestCasePlatform.m

```diff
@@ -34,7 +34,7 @@ BOOL FBSnapshotTestCaseIs64Bit(void)
 NSString *FBDeviceAgnosticNormalizedFileName(NSString *fileName)
 {
   UIDevice *device = [UIDevice currentDevice];
-  CGSize screenSize = [[UIApplication sharedApplication] keyWindow].bounds.size;
```

Comment:
hmm, we could check if the `keyWindow` is `nil` and if it is, create a window with the `[[UIScreen mainScreen] bounds]`

How does this sound?